extensions/vscode/src/views/homeView.ts

```diff
@@ -286,12 +280,6 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
   }
 
   private async updateFileList(uri: string, action: FileAction) {
-    const activeDeployment = this.getActiveContentRecord();
-    if (!activeDeployment) {
-      console.error("homeView::updateFileList: No active deployment.");
-      return;
-    }
-    // this will only be true if the config really exists
```

Comment:
A lot of cases like this will never occur because of the active config being so tied to the active deployment, but we don't need to rely on the early termination since these calls all still work if we have a mistake where we have an active config but no active deployment.